src/sidebar/util/test/oauth-client-test.js

```diff
@@ -192,7 +191,7 @@ describe('sidebar/util/oauth-client', () => {
         fakeWindow.open,
         'about:blank',
         'Log in to Hypothesis',
-        'height=430,left=274.5,top=169,width=475'
+        'left=274.5,top=169,width=475,height=430'
```

Comment:
These parameters used to be sorted in alphabetical order, but that no longer happens.